components/filter/filter.js

```diff
@@ -590,10 +590,13 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 		this._activeFiltersSubscribers.updateSubscribers();
 	}
 
-	_dispatchDimensionFirstOpenEvent(key) {
-		if (!this._openedDimensions.includes(key)) {
-			this.dispatchEvent(new CustomEvent('d2l-filter-dimension-first-open', { bubbles: true, composed: false, detail: { key: key } }));
-			this._openedDimensions.push(key);
+	_dispatchDimensionFirstOpenEvent(dimension) {
+		if (!this._openedDimensions.includes(dimension.key)) {
+			if (dimension.searchType === 'manual') {
+				this._search(dimension);
+			}
+			this.dispatchEvent(new CustomEvent('d2l-filter-dimension-first-open', { bubbles: true, composed: false, detail: { key: dimension.key } }));
```

Comment:
I wonder if the order matters here 🤔. I think the main point of this event is for consumers to wait to load the options until they get this event. Would they rather get the opened event and _then_ the search event, since searching before loading the options doesn't really make sense?

Reply:
Got it, yeah that is logical. I didn't know that was the purpose of the event

Reply:
I still think they'll need to handle it properly on their side if they're using both, since loading the options is usually `async`.  But I _think_ that order will be more intuitive to do so